pages/organizations/[oid]/information.vue

```diff
@@ -0,0 +1,233 @@
+<template>
+  <div>
+    <div class="flex flex-wrap">
+      <h2 class="!text-sm !mb-2.5 w-full flex-none sm:flex-1">
+        {{ $t('Statistics from the last 12 months') }}
+      </h2>
+      <div>
+        <BrandedButton
```

Comment:
I think we're missing some margin around this button?
Since we don't have all the stat boxes